src/aspire/numeric/complex_pca/complex_pca.py

```diff
@@ -45,6 +48,8 @@ def _fit(self, X):
             allow_complex=True,
         )
 
+        xp, is_array_api_compliant = get_namespace(X)
```

Comment:
Just want to be sure that you expect this command to work with the supported versions of scikit.  If it doesn't work with earlier versions then it could use a branch for safety.

Reply:
This command is used with all supported versions of scikit. The changes made by scikit involved moving the call to this function out of the methods `_fit_full` and `_fit_truncated` into the higher level method `_fit` that calls them. So I made the same change here in our subclass.

Reply:
> This command is used with all supported versions of scikit.

Awesome, all good then! Thanks! 🚀